src/fiatconnect/KycStatusScreen.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
```

Comment:
Added this in as a placeholder to have a unique place to navigate to from the quote handler saga.